test/functional/cmdLineTests/gcRegressionTests/gcRegressionTests.xml

```diff
@@ -223,7 +223,14 @@
  <test id="Excessive GC throws OOM">
  	<echo value="CMVC 148977:  Excessive GC throws OOM may fail if there is other activity on the machine - at least 5 GCs are required, in the 20 second test" />
  	<!-- does the test fail to run (thus implying that we encountered excessive GC)? -->
- 	<command>$EXE$ $XINT$ $ARGS_FOR_ALL_TESTS$ -Xdump:none -Xms8m -Xmx8m -Xgc:fvtest=forceExcessiveAllocFailureAfter=5 -verbose:gc -Xverbosegclog:foo.log $CP$ com.ibm.tests.garbagecollector.SpinAllocate 20</command>
+ 	<command>$EXE$ $XINT$ $ARGS_FOR_ALL_TESTS$ -Xdump:none -Xms2m -Xmx2m -Xgc:fvtest=forceExcessiveAllocFailureAfter=5 -verbose:gc -Xverbosegclog:foo.log $CP$ com.ibm.tests.garbagecollector.SpinAllocate 20</command>
```

Comment:
Do you need this change? I thought we agreed to keep `-Xms8m -Xmx8m` for all non-RISC-V cases

Reply:
Just restored it back to `-Xms8m -Xmx8m` as confirmed previously for all non-RISC-V cases.